### PR TITLE
swi-prolog-devel: Updated to version 8.3.20

### PR DESCRIPTION
**Pull request by JanWielemaker**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 11.2.1 20D74
Xcode 12.4 12D4e

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: lang/swi-prolog-devel/Portfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ PortGroup           cmake 1.1
 name                swi-prolog-devel
 conflicts           swi-prolog swi-prolog-lite
 epoch               20051223
-version             8.3.19
+version             8.3.20
 revision            1
```
Comment (reneeotten) [inline lang/swi-prolog-devel/Portfile:10]: if you update the `version` then you should reset the `revision` to `0`

Comment (JanWielemaker) [reply lang/swi-prolog-devel/Portfile:10]: Thanks.  Will try to update on next release.

Comment (reneeotten) [reply lang/swi-prolog-devel/Portfile:10]: that should ideally be done before merging this PR

Comment (JanWielemaker) [reply lang/swi-prolog-devel/Portfile]: Ah, it is not merged yet.  I pushed a forced update for this PR after rebase to current master and changing 1 to 0.  Hope this works.   Feel free to make minor edits.   Subsequent releases pull the current version and edit the version and hashes using a script, so enhancements are kept.

Comment (reneeotten) [reply lang/swi-prolog-devel/Portfile]: thanks!

**Pull request merged**